### DISCUSSION TITLE
Can't login to Github with Firefox

### DISCUSSION THREAD

**Discussion by SillyStrategy** [community/community#120882] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

- Fresh Win11 install
- Using Firefox v125.0.1
- Firefox configured to always use private browsing mode
- Firefox tracking protection set to "strict"
- Using Quad9 as DNS over HTTPS provider (https://dns9.quad9.net/dns-query)
- Only installed extension is Bitwarden password manager

Problem is only present with Firefox, Edge and Chrome work fine.
I've changed the settings above to "stock" (no private browsing mode, standard tracking protection and no DNS over HTTPS) and problem persists.

Please help! I don't like to use Edge or Chrome at all, and would like to continue using Firefox as my default browser.

Whenever I try to login using a username/password/2fa combo I get redirected to a 404 page that's supposed to be the 2FA input screen
![image](https://github.com/community/community/assets/147859569/d367f461-b9b6-4772-9705-251d64aeaa6b)

Whenever I try to login using passkeys, I get a generic error message saying login is not possible
![image](https://github.com/community/community/assets/147859569/efad843d-82af-41e1-9848-9d009deb1457)

**Comment by ghost** [comment]
Steps to Resolve:
Disable Private Browsing Mode and Tracking Protection:
Open Firefox and go to Menu > Options > Privacy & Security.
Turn off private browsing mode and set tracking protection to "Standard".
Restart Firefox and try logging in again to see if the issue persists.

Clear Firefox Cache and Cookies:
Clearing cached data and cookies can often resolve login issues.
Go to Menu > Options > Privacy & Security > Cookies and Site Data.
Click on 'Clear Data' and restart Firefox.

Check for Browser Updates:
Ensure Firefox is up to date by going to Menu > Help > About Firefox.
Update to the latest version if available and test the login again.

Verify Windows 11 System Settings:
Confirm that your Windows 11 system is fully updated.
Check for any system-wide network settings or firewalls that could be interfering with Firefox.

Test in Safe Mode:
Run Firefox in Safe Mode to rule out conflicts caused by settings or extensions.
Press Shift while launching Firefox or go to Menu > Help > Restart with Add-ons Disabled.
Try logging in during Safe Mode to see if the issue persists.

Monitor Console Logs for Errors:
Open Firefox and press Ctrl+Shift+J to view the browser console.
Look for any error messages or warnings that may provide insights into the login issue.

**Reply by SillyStrategy** [reply]
Thank you for your reply. As I mentioned in my post I'm already implementing the steps you have provided.
- I'm using the latest Firefox browser version available to date (v125.0.1)
- Since I'm using "always private mode", there is no cache or cookies to delete
- I'm using a clean windows 11 install with all windows update patches applied
- The only extension installed is the Bitwarden password manager extension (to handle passkeys)
- For several reasons I won't disable private browsing mode or strict privacy settings

This was not a problem before, what might have changed from the Github side? 😒

**Comment by arefiev** [comment]
Same issue here.  I cannot log into Github with Firefox in private mode, so I am forced to use the non-private mode.  It worked before but does not anymore sadly
